OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidEntryPointsFinder.scala

```diff
@@ -0,0 +1,52 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package cg
+package android
+
+import scala.collection.mutable.ArrayBuffer
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+import org.opalj.br.Method
+import org.opalj.br.MethodDescriptor
+import org.opalj.br.ReferenceType
+import org.opalj.br.analyses.SomeProject
+import org.opalj.br.analyses.cg.EntryPointFinder
+import org.opalj.tac.fpcf.analyses.MethodDescription
+
+/**
+ * The AndroidEntryPointFinder considers specific methods of launcher Activity Clases as entry points.
+ * An activity is a launcher activity if it contains an intent filter with action "android.intent.action.MAIN"
+ * and category "android.intent.category.LAUNCHER". Requires Android Manifest to be loaded.
```

Comment:
this is out of scope: #223

Reply:
Hm, kind of. Still, this lets this PR end up with a ProjectInformationKey that doesn't abide by the rules for ProjectInformationKeys. This can end up in deadlocks if users of the key aren't aware of it.

Reply:
Can we please fix this first? If it is out of scope here, make it another PR and let us merge that first. I really don't like this dependency given in a ScalaDoc...